component/class/defaults.yml

```diff
@@ -1,11 +1,13 @@
 parameters:
   cloudscale_metrics_collector:
     secrets:
-      cloudscale:
+      credentials:
         stringData:
-          token: "?{vaultkv:${cluster:tenant}/${cluster:name}/cloudscale-metrics-collector/token}"
+          CLOUDSCALE_API_TOKEN: "?{vaultkv:${cluster:tenant}/${cluster:name}/cloudscale-metrics-collector/token}"
+          KUBERNETES_SERVER_URL: "?{vaultkv:${cluster:tenant}/${cluster:name}/cloudscale-metrics-collector/${_instance}/cluster-server}"
+          KUBERNETES_SERVER_TOKEN: "?{vaultkv:${cluster:tenant}/${cluster:name}/cloudscale-metrics-collector/${_instance}/cluster-token}"
```

Comment:
What do you mean? Switching from `token` to `CLOUDSCALE_API_TOKEN`? or something else

Reply:
Both switching to `credentials` form `cloudscale` and `token` to `CLOUDSCALE_API_TOKEN` are technically breaking API changes for anyone using this component. 

I.e. if they set anything for the `token` this will break. 

I don't think we did this anywhere so for this it's just me being nitpicky and you can also just ignore me :)

Reply:
ah I see. Yeah that's true. But since it's semver 0.x breaking changes are ok, plus you'd need to actually update the component to the next version in order for it to break. Which we'll do for our use cases anyway.